src/models/auth/Client.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose')
+
+const clientSchema = mongoose.Schema(
+  {
+    id: String, // client_id
```

Comment:
yeah, tbh I'm not sure if it's needed, I feel like it might be due to the oauth module needing an id field specifically so maybe _id won't work, but I couldn't really test this at the time